angularjs-portal-home/src/main/webapp/samples/notifications.json

```diff
@@ -21,6 +21,14 @@
         "title"  : "You need to modify your NetID account to activate essential UW Services.",
         "actionURL" : "https://www.mynetid.wisc.edu/modify",
         "actionAlt" : "Activate Services"
+      },
+      {
+        "id"     : 4,
+        "groups" : ["Facstaff","Portal Administrators"],
+        "title"  : "Employee benefits open enrollment for 2016 starts Oct 5th, 2015 and ends Oct 30th, 2015.",
```

Comment:
Further tweak:

> UW Employee Annual Benefits Enrollment starts October 5 and ends October 30, 2015

I suggest, tweaking the tweak: ending with a period to make it a sentence.